anyway/views/schools/api.py

```diff
@@ -17,7 +17,7 @@ def schools_api():
         db.session.query(School)
         .filter(
             not_(and_(School.latitude == 0, School.longitude == 0)),
-            not_(and_(School.latitude == None, School.longitude == None)),
+            not_(and_(School.latitude is not None, School.longitude is not None)),
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969